src/admin-views/editor/panel/ticket.php

```diff
@@ -189,7 +216,132 @@ class="ticket_field ticket_provider"
 				 */
 				do_action( 'tribe_events_tickets_metabox_edit_main', $post_id, $ticket_id ); ?>
 			</section>
-
+			<div class="input_block">
+				<label class="ticket_form_label ticket_form_left" for="ticket_description"><?php esc_html_e( 'Description:', 'event-tickets' ); ?></label>
+				<textarea
+						rows="5"
+						cols="40"
+						name="ticket_description"
+						class="ticket_field ticket_form_right"
+						id="ticket_description"
+				><?php echo esc_textarea( $ticket ? $ticket->description : null ) ?></textarea>
+				<div class="input_block">
+					<label class="tribe_soft_note">
+						<input
+								type="checkbox"
+								id="tribe_tickets_show_description"
+								name="ticket_show_description"
+								value="1"
+								class="ticket_field ticket_form_left"
+								<?php checked( true, $ticket ? $ticket->show_description : true ); ?>
+						>
+						<?php
+						echo esc_html( sprintf(
+								__( 'Show description on front end %s form.', 'event-tickets' ),
```

Comment:
* :no_entry_sign: Multi-line function call not indented correctly; expected 28 spaces but found 32
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: a3df82054655c6bf75c929ee3f51168b</sub>